content/08.outlook.md

```diff
@@ -18,11 +18,11 @@ While many techniques exist to collate rare data from different sources, low-qua
 In our experience, collaboration with domain experts has proved to be critical in gaining insight into potential sources of variation in the datasets.
 An anecdotal example: conversations with a clinician revealed that samples in a particular tumor dataset were collected using vastly different surgical techniques (laser ablation and excision vs standard excision).
 This information, not readily available to non-experts, was obvious to the clinician.
-Such instances suggest that collaboration with domain experts and sharing of well-annotated data is needed to generate robust datasets in the future.
+Such instances suggest that collaboration with domain experts and sharing of well-annotated data is needed to generate robust datasets.
 
 In addition to sample scarcity, comprehensive phenotypic-genotypic databases are also lacking.
-Rare disease studies that collect genomic and phenotypic data are becoming more common [@doi:10.1038/nrg3555; @doi:10.1038/nrg.2017.116; @doi:10.1056/NEJMra1711801].
-Developing comprehensive genomics-based genotype-phenotype databases that prioritize clinical and genomics data standards is key to fuel interpretation of features extracted using ML methods, possibly by funding or fostering collaboration between biobanking projects and patient registry initiatives.
+While rare disease studies that collect genomic and phenotypic data are becoming more common [@doi:10.1038/nrg3555; @doi:10.1038/nrg.2017.116; @doi:10.1056/NEJMra1711801], developing comprehensive genomics-based genotype-phenotype databases that prioritize clinical and genomics data standards is key to fueling interpretation of ML methods.
+This method can be bolstered by funding or fostering collaboration between biobanking projects and patient registry initiatives.
 Mindful sharing of data with proper metadata and attribution enabling prompt data reuse is important in building valuable datasets for rare disease research [@https://www.nature.com/articles/s41576-020-0257-5].
 Finally, federated learning methods, such as those used in mobile health [@doi:10.1038/s41746-020-00323-1] and electronic healthcare records studies [@doi:10.1001/jamanetworkopen.2021.24946], may allow researchers to develop ML models on data from larger numbers of people with rare diseases whilst protecting patient privacy.
```

Comment:
Re: line 29

Justin: The paragraphs that follow this heading don’t really reflect this heading, i.e. don’t focus on mechanistic understanding.

Reply:
Re: line 33 `can handle constraints of rare disease data`

Justin: this is not clear